addon/serializers/json-api.js

```diff
@@ -391,6 +374,12 @@ const JSONAPISerializer = JSONSerializer.extend({
   },
 
   /**
+    Dasherizes and singularizes the model name in the payload to match
+    the format Ember Data uses internally for the model name.
+
+    For Example: `posts` => `post`, `studentAssesments` =>
```

Comment:
Actually, that might not makes much sense. Could you instead write it out in prose? the `=>` is a bit confusing as it's also the syntax for arrow functions.